src/containers/profile/ProfileSections/AccountSettings/AccountSettingsSections/ProfileInfo/ProfileInfo.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { connect } from "react-redux";
+import "./ProfileInfo.css";
+import profileInfoEditButtonIcon from "../../../../../../images/editButton.png";
+import ShowProfileInfo from "./ProfileInfoSections/ShowProfileInfo";
+import EditProfileInfo from "./ProfileInfoSections/EditProfileInfo";
+import { Modal, Button } from "react-bootstrap";
+
+const UserInfo = (props) => {
+  const { id, authError, errMessage, errKey } = props;
+  const [isEditing, setIsEditing] = React.useState(false);
+  const [
+    profileInfoEditButtonOpacity,
+    setProfileInfoEditButtonOpacity,
+  ] = React.useState("1");
+  const [successModalShow, setSuccessModalShow] = React.useState(false);
+
+  const profileInfoEditButton = () => {
+    return (
+      <img
+        alt="Edit button"
+        src={profileInfoEditButtonIcon}
+        className="profileInfoEditButton"
+        onClick={() => {
+          setIsEditing(true);
+          setProfileInfoEditButtonOpacity("0.2");
+        }}
+        style={{ opacity: profileInfoEditButtonOpacity }}
+      />
+    );
+  };
+
+  // Handle personal info editing
+  const handleEditingStatus = (isEdited) => {
+    setIsEditing(false);
+    setProfileInfoEditButtonOpacity("1");
```

Comment:
generally we do not mix css in the code in this way. instead you should add/remove a class if you want to use JS and have the class add/remove opacity